src/civetweb.c

```diff
@@ -726,6 +730,9 @@ static struct mg_option config_options[] = {
     {"ssi_pattern",                 CONFIG_TYPE_EXT_PATTERN,   "**.shtml$|**.shtm$"},
     {"throttle",                    12345,                     NULL},
     {"access_log_file",             CONFIG_TYPE_FILE,          NULL},
+    {"access_log_format",           CONFIG_TYPE_STRING,        "c-ip cs-username date time cs-method "
+															   "cs-uri sc-status sc-bytes "
```

Comment:
Can this be re-aligned ? The 2 lines here are not visible on github, but visible on explicit cmd-line diff